clang/runtime/dpct-rt/include/device.hpp.inc

```diff
@@ -269,6 +269,22 @@ public:
 // DPCT_DEPENDENCY_END
 // DPCT_CODE
   size_t get_local_mem_size() const { return _local_mem_size; }
+// DPCT_LABEL_END
+// DPCT_LABEL_BEGIN|device_info_get_memory_clock_rate|dpct
+// DPCT_PARENT_FEATURE|device_info
```

Comment:
Zhiwei's email says that: The “DPCT_PARENT_FEATURE” annotation is not recommended to use now.